application/ui/src/features/inspect/dataset/media-preview/inference-result/inference-result.component.tsx

```diff
@@ -17,49 +18,65 @@ interface InferenceResultProps {
     inferenceResult: SchemaPredictionResponse | undefined;
 }
 
+const labelHeight: Responsive<DimensionValue> = 'size-350';
+
 export const InferenceResult = ({ selectedMediaItem, inferenceResult }: InferenceResultProps) => {
+    const imageRef = useRef(null);
     const { inferenceOpacity } = useInference();
-    const [isVerticalImage, setIsVerticalImage] = useState(false);
+    const [imageDimensions, setImageDimensions] = useState({ width: 0, height: 0, left: 0, top: 0 });
```

Comment:
Shouldn't we be able to set the dimensions based on `selectedMediaItem`? I would expect that width and height is available from the media items endpoint.

We could then use the same `ZoomTransform` component to make this component fit into its container's width / height.